templates/login_post.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * SPDX-FileCopyrightText: 2016 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+/** @var array $_ */
+?>
+Please wait while you are redirected to the SSO server.
+
+<form action="<?= $_['ssoUrl'] ?>" method="post">
+	<input type="hidden" name="SAMLRequest" value="<?= $_['samlRequest'] ?>" />
+	<input type="hidden" name="RelayState" value="<?= $_['relayState'] ?>" />
+	<input type="hidden" name="SigAlg" value="<?= $_['sigAlg'] ?>" />
+	<input type="hidden" name="Signature" value="<?= $_['signature'] ?>" />
+	<noscript>
+		<p>JavaScript is disabled. Click the button below to continue.</p>
```

Comment:
Should be translatable. The I18N instance is always available in templates vias `$l`.

```suggestion
		<p><?php p($l->t('JavaScript is disabled. Click the button below to continue.')) ?></p>
```